src/main/java/qna/service/QnAService.java

```diff
@@ -20,9 +17,6 @@
     @Resource(name = "questionRepository")
     private QuestionRepository questionRepository;
```

Comment:
@Resource를 사용하면 빈 검색 시 우선순위가 name인걸로 알고있는데요, service name을 지정하고 name을 기준으로 찾는 이유가 뭔가요? 보통은 아래처럼 사용하면 편할텐데 어떤경우때문인지 모르겠네요.
```suggestion
@Service
public class QnAService {
    @Autowired
    private QuestionRepository questionRepository;
```

Reply:
음.. 이 부분은 왜 이렇게 사용했는지 저도 의도를 모르겠네요 ㅎㅎ 차이가 있다면, Resource의 경우 java에서 자체적으로 주입해주는 것이고, @Autowired의 경우 spring에서 주입해주는 정도인 것 같습니다.
크게 신경쓰지 않아도 될 것 같아요!